### PR TITLE
auth0 example

### PR DESCRIPTION
**Pull request by AldoSGNL**
Hi Marc. I'm adding the Auth0 example. Please review and merge it when you have a chance. Thank you.

### CODE REVIEW

File: auth0/login-action.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  
+  //Create an instance of Axios HTTP client
+
+  const sgnl_api = axios.create({
+  baseURL: event.secrets.sgnl_url,
+  timeout: 1000,
+  headers: {'Authorization': event.secrets.Token}
+  });
+
+  //Output requested resource to log for debug
```
Comment (marc-sgnl) [inline auth0/login-action.js:18]:
```suggestion
  // Output requested resource to log for debug
```

File: auth0/login-action.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+})
+  .then(function (response) {
+    console.log("SGNL policy decision: " + response.data.decisions[0].decision)
+   
+    //Deny access if SGNL policy results in a deny.
+    if (response.data.decisions[0].decision == "Deny") {
+    api.access.deny(`SGNL Authorization: Access to ${event.client.name} is not allowed.`);
+    api.access.deny
+  } else {
+    //All good.
```
Comment (marc-sgnl) [inline auth0/login-action.js:39]:
```suggestion
    // Success
```

**Review by marc-sgnl — APPROVED** [review]
Some nitpicks, but nothing blocking.

**Comment by AldoSGNL** [comment]
@marc-sgnl We should be good now. Please review and approve the sample request context JSON. Thanks!

**Review by marc-sgnl — APPROVED** [review]

**Pull request merged**